### PR TITLE
ci: Fix build in databend-docker

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

I hereby agree to the terms of the CLA available at: https://databend.rs/policies/cla/

## Summary

This PR is an extension of https://github.com/datafuselabs/databend/pull/3391, fix an extra place of cross.

## Changelog

- Build/Testing/CI

### CODE REVIEW

File: .github/workflows/databend-release.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Rust setup
         run: |
           bash ./scripts/setup/dev_setup.sh
-          cargo install --version 0.2.1 cross
+          cargo install cross
```
Comment (ZhiHanZ) [inline .github/workflows/databend-release.yml:35]: I think we should keep exact version number to get avoid of surprising breaking change

File: .github/workflows/databend-docker.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
       - name: Rust setup
         run: |
           bash ./scripts/setup/dev_setup.sh
-          cargo install --version 0.1.16 cross
+          cargo install cross
```
Comment (Xuanwo) [inline .github/workflows/databend-docker.yml:48]:
```suggestion
          cargo install --version 0.2.1 cross
```

File: .github/workflows/databend-release.yml (hunk 1 above)
Comment (Xuanwo) [reply .github/workflows/databend-release.yml:35]:
```suggestion
          cargo install --version 0.2.1 cross
```

Comment (Xuanwo) [reply .github/workflows/databend-release.yml]: Makes sense, addressed.

**Review by ZhiHanZ — APPROVED** [review]

**Comment by BohuTANG** [comment]
Build perf tool failed, seems related to OpenSSL :

https://github.com/datafuselabs/databend/runs/4496828786?check_suite_focus=true

```
run pkg_config fail: "pkg-config has not been configured to support cross-compilation.\n\nInstall a sysroot for the target platform and configure it via\nPKG_CONFIG_SYSROOT_DIR and PKG_CONFIG_PATH, or install a\ncross-compiling wrapper for pkg-config and set it via\nPKG_CONFIG environment variable."

  --- stderr
  thread 'main' panicked at '

  Could not find directory of OpenSSL installation, and this `-sys` crate cannot
  proceed without this knowledge. If OpenSSL is installed and this crate had
  trouble finding it,  you can set the `OPENSSL_DIR` environment variable for the
  compilation process.

  Make sure you also have the development packages of openssl installed.
  For example, `libssl-dev` on Ubuntu or `openssl-devel` on Fedora.

  If you're in a situation where you think the directory *should* be found
  automatically, please open a bug at https://github.com/sfackler/rust-openssl
  and include information about your system as well as this message.

  $HOST = x86_64-unknown-linux-gnu
  $TARGET = aarch64-unknown-linux-gnu
  openssl-sys = 0.9.71

  ', /cargo/registry/src/github.com-1ecc6299db9ec823/openssl-sys-0.9.71/build/find_normal.rs:180:5
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
Error: Process completed with exit code 101.


```

**Comment by Xuanwo** [comment]
Wired,  I will figure it out tomorrow.

**Comment by Xuanwo** [comment]
The problem is gone on the `main` branch, I believe there is something wrong with the `~/.cargo/bin` cache that failed workflows use the wrong `cross` binary.

Now all the build is OK and I'm working https://github.com/datafuselabs/databend/pull/3344 so that we can avoid those problems again.

**Pull request merged**